app/src/main/res/values/strings.xml

```diff
@@ -596,4 +595,24 @@
     <plurals name="dynamic_seek_duration_description">
         <item quantity="other">%s seconds</item>
     </plurals>
-</resources>
+
+    <!-- Feed -->
+    <string name="fragment_feed_title">What\'s New</string>
+    <string name="feed_groups_header_title">Feed groups</string>
+    <string name="feed_oldest_subscription_update">Oldest subscription update: %s</string>
+    <string name="feed_subscription_not_loaded_count">Not loaded: %d</string>
+    <string name="feed_notification_loading">Loading feed…</string>
+    <string name="feed_processing_message">Processing feed…</string>
+    <string name="feed_group_dialog_select_subscriptions">Select Subscriptions</string>
+    <string name="feed_group_dialog_selection_count">Selected: %d</string>
+    <string name="feed_group_dialog_empty_selection">No subscriptions selected</string>
+    <string name="feed_group_dialog_empty_name">Name of the group is empty</string>
+    <string name="feed_group_dialog_name_input">Name</string>
+    <string name="feed_create_new_group_button_title">New</string>
+
+    <string name="settings_category_feed_title">Feed</string>
+    <string name="feed_update_threshold_title">Feed update threshold</string>
+    <string name="feed_update_threshold_summary">How long has it been since the last update before a subscription is considered outdated. — %s </string>
```

Comment:
I'm not sure about this string, it could be improved but I'm not sure how

Reply:
Maybe "Time after last update before a subscription is considered outdated - %s"
or "A feed is outdated %s after the last update"

Reply:
I chose the first suggestion from @TobiGr. If someone can come up with something even more understandable, you're welcome to tell it here.